docs/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,10 +15,11 @@ Check off the items that are true.
 - [ ] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
 - [ ] The game runs without errors. 
 - [ ] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.

```

Comment:
```suggestion
- [ ] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s. (No spaces or other special characters.)
```
This might be helpful to clarify ot a bit.